src/sssom/util.py

```diff
@@ -504,17 +504,40 @@ def compare_dataframes(df1: pd.DataFrame, df2: pd.DataFrame) -> MappingSetDiff:
     return d
 
 
-def dataframe_to_ptable(df: pd.DataFrame, *, inverse_factor: float = None):
+def add_default_confidence(df: pd.DataFrame, confidence: float = None) -> pd.DataFrame:
+    """Add `confidence` column to DataFrame if absent and initializes to 0.95.
+
+    If `confidence` column already exists, only fill in the None ones by 0.95.
+
+    :param df: DataFrame whose `confidence` column needs to be filled.
+    :return: DataFrame with a complete `confidence` column.
+    """
+    if CONFIDENCE in df.columns:
+        df[CONFIDENCE] = confidence * df.get(CONFIDENCE, 1)
```

Comment:
What happens if there is a 

NaN (a non numeric string)
None or ""
Or NULL

In the `confidence` column?

Reply:
Good catch! Fixed it